go-controller/pkg/cni/cni.go

```diff
@@ -73,6 +77,12 @@ func (pr *PodRequest) cmdAdd() *PodResult {
 	}
 	kubecli := &kube.Kube{KClient: clientset}
 
+	isDefaultInterface := networkName == "ovn-kubernetes"
```

Comment:
this "is default" check is done in two places. probably better to pout in a helper function

Reply:
see: https://github.com/AlonaKaplan/ovn-kubernetes/pull/3/files#diff-0f28a8316299be5031f7bf5ec31b441aR115

Reply:
done